packages/phoenix-evals/src/phoenix/evals/functions/processing.py

```diff
@@ -0,0 +1,138 @@
+"""
+Token processing functions for supported models. This module is being deprecated.
+"""
+import logging
+import sys
+from typing import Any, List
+
+import tiktoken
+
+from ..models import BaseModel
+
+logger = logging.getLogger(__name__)
+
+OPENAI_MODEL_TOKEN_LIMIT_MAPPING = {
+    "gpt-3.5-turbo-instruct": 4096,
+    "gpt-3.5-turbo-0301": 4096,
+    "gpt-3.5-turbo-0613": 4096,  # Current gpt-3.5-turbo default
+    "gpt-3.5-turbo-16k-0613": 16385,
+    "gpt-4-0314": 8192,
+    "gpt-4-0613": 8192,  # Current gpt-4 default
+    "gpt-4-32k-0314": 32768,
+    "gpt-4-32k-0613": 32768,
+    "gpt-4-1106-preview": 128000,
+    "gpt-4-vision-preview": 128000,
+}
```

Comment:
bit confused - wasn't the idea to nix this?

Reply:
All this behavior is now isolated to the processing module, let's defer moving this fully into the scripts directory after the release?